frontend/src/lib/utils.tsx

```diff
@@ -602,7 +602,7 @@ export function colonDelimitedDuration(d: string | number | null | undefined, nu
         m = Math.floor(s / 60)
         s -= m * 60
     }
-    s = Math.round(s)
+    s = Math.floor(s)
```

Comment:
Snuck in a small bug fix. This could return `00:00:60` when it should be `00:00:59`